src/backend/base/langflow/base/agents/events.py

```
@@ -191,13 +191,37 @@
 
     if tool_content and isinstance(tool_content, ToolContent):
-        tool_content.output = event["data"].get("output")
-        duration = _calculate_duration(start_time)
-        tool_content.duration = duration
-        tool_content.header = {"title": f"Executed **{tool_content.name}**", "icon": "Hammer"}
-
+        # Call send_message_method first to get the updated message structure
         agent_message = await send_message_method(message=agent_message)
-        new_start_time = perf_counter()  # Get new start time for next operation
+        new_start_time = perf_counter()
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)